app/retail/templates/shared/grant_thanks_modal.html

```diff
@@ -42,16 +42,12 @@ <h5>Thanks for Contributing! You Earned a Kudos</h5>
               <a class="button button--primary" role="button" href="{{fund_reward.url}}?tweet=I just funded a grant on @gitcoin (https://{{ request.META.HTTP_HOST }}{% url 'grants:details' grant.id grant.slug %}) and got this sweet Kudos! => "
                 target="_blank" rel="noopener noreferrer" >
                 {% trans "Redeem Kudos" %} 🌈
-              </a> |
-            {% elif grant.id %}
-              <a class="button button--primary" role="button" href="https://twitter.com/intent/tweet?text=I just funded this grant on @gitcoin => https://{{ request.META.HTTP_HOST }}/{% url 'grants:details' grant.id grant.slug %}"
-                target="_blank" rel="noopener noreferrer" OnClick="ga('send', 'event', 'Tweet Button', 'click', 'Grant Funder')">
-                {% trans "Tweet About this Grant" %} 🐦
-              </a>
+              </a> 
             {% else %}
-              <a class="button button--primary" role="button" href="https://twitter.com/intent/tweet?text=I just funded grants on @gitcoin => https://{{ request.META.HTTP_HOST }}/grants"
-                target="_blank" rel="noopener noreferrer" OnClick="ga('send', 'event', 'Tweet Button', 'click', 'Grant Funder')">
-                {% trans "Tweet About It" %} 🐦
+              <a class="button button--primary" role="button"
```

Comment:
Was  the tweet button intentionally removed ?

Reply:
@PixelantDesign we probably want both the tweet button as well as the fund a new grant button, correct?

Reply:
mmm yes tweet seems was something people like. Unless @PixelantDesign asked to remove this, lets get it back.

Reply:
Done!